src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -154,6 +155,33 @@ private static String getMessageForConstraintViolation(final ConstraintViolation
     return ResponseEntity.ok(collectionExerciseSummaryDTOList);
   }
 
+  /**
+   * GET to find collection exercises for the surveys in the given list of survey ids.
+   *
+   * @param surveyIds survey Ids for which to get collection exercises
```

Comment:
This comment doesn't read very well.  Can you rewrite it?

Reply:
Changed